azurelinuxagent/common/conf.py

```diff
@@ -80,6 +80,9 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
         raise AgentConfigError(("Failed to load conf file:{0}, {1}"
                                 "").format(conf_file_path, err))
 
+def enable_rdma(conf=__conf__):
+    return conf.get_switch("OS.EnableRDMA", False)
```

Comment:
so this is a new key, can you please update the `waagent.conf` and the `README` please.

it also means we are removing support for:

* OS.CheckRdmaDriver 
* OS.UpdateRdmaDriver
* OS.RdmaRepository
* OS.PackageExtraConfig

keys as well. Therefore we should update the waagent.conf and the README ideally.